sharktank/tests/ops/sharded_test.py

```diff
@@ -295,6 +299,26 @@ def testCatNonSplitDim(self):
         actual_result = ops.cat([sharded_a, sharded_b], dim=cat_dim)
         assert expected_result.is_deep_equal(actual_result, compare_name=False)
 
+    def testCatReplicatedShouldFail(self):
```

Comment:
This probably belongs in `pipeline_parallelized_test.py` since that's where all the pp-related ops tests currently are.

Reply:
I moved it.